modules/dnn/test/test_tf_importer.cpp

```diff
@@ -1123,4 +1123,37 @@ TEST_P(Test_TensorFlow_nets, Mask_RCNN)
         expectNoFallbacks(net);
 }
 
+TEST_P(Test_TensorFlow_nets, EfficientDet)
+{
+    if (target != DNN_TARGET_CPU)
+    {
+        if (target == DNN_TARGET_OPENCL_FP16) applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_OPENCL_FP16);
+        if (target == DNN_TARGET_OPENCL)      applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_OPENCL);
+        if (target == DNN_TARGET_MYRIAD)      applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_MYRIAD);
+    }
```

Comment:
Is test skipped for `OCV` and` DLIE` backends or only for `DLIE`?

Reply:
For OpenCV as well

Reply:
I think it is better to use  `CV_TEST_TAG_DNN_SKIP_OPENCL` and `CV_TEST_TAG_DNN_SKIP_OPENCL_FP16`